File: python/mario.py
from cs50 import get_int
# get input from the user
while True:
    n = get_int("Height:")
    # cheking if int is in range
    if n in range(1, 9):
        break
m = n + 1
# printing each row 
for i in range(1, m):
    # print spaces
    print(" " * (n - i), end='')
    # print hashes
    print("#" * i, end='')
    # print two sapces
    print("  ", end='')
    # print hashes
    print("#" * i, end='')
    # printing new line
    print()

File: c/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do 
    {
        height = get_int("height:"); 
    }
    while (height < 1 || height > 8);        
    // each line
    for (int j = 0; j < height; j++)
    {
//spaces
        for (int i = 1; i <= height - (j + 1); i++)
        {
            printf(" ");
        }
//hastags    
        for (int k = 0; k <= j; k++)
        {
            printf("#");
        }    
//two spaces
        printf("  ");
//hastags
        for (int k = 0; k <= j; k++)
        {
            printf("#");
        }
//new line
        printf("\n");
    }
}
